modules/distr-tracing-tempo-accessing-console.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-accessing-console_{context}"]
+= Accessing the Tempo console
+
+To access the Tempo console you must have either {SMProductName} or {DTProductName} installed, and {TempoName} installed, configured, and deployed.
+
+The installation process creates a route to access the Tempo console.
+
+If you know the URL for the Tempo console, you can access it directly. If you do not know the URL, use the following directions.
+
+.Procedure from OpenShift console
+. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Navigate to *Networking* -> *Routes*.
+
+. On the *Routes* page, select the control plane project, for example `tracing-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+. If necessary, use the filter to find the `tempo` route. Click the route *Location* to launch the console.
+
+. Click *Log In With OpenShift*.
+
+////
+.Procedure from Kiali console
```

Comment:
I think we can remove it from tempo docs.

Reply:
Thank you, Ruben. Applying.